src/store/groups.js

```diff
@@ -106,22 +129,22 @@ const mutations = {
 				group.contacts.splice(index, 1)
 			}
 		})
+		if (state.orderGroups === 'amount') {
+			state.sortedGroups = [...state.groups].sort(sortFunc.amount)
```

Comment:
this line is weird because i dont map it, but there is no reason for mapping because i dont need to add toString to each object

Reply:
ah sure, you can also do a clone with slice
`state.groups.slice().sort(sortFunc.amount)`